### PR TITLE
fix(test) : fix test to run against production and prod-preview

### PR DESCRIPTION
**Pull request by Raunak1203**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [x] What does this PR do? 
This PR fixes the tests to run against production and on prod-preview
- [ ] What issue/task does this PR references?

- [x] Are the tests Included?
yes
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by Raunak1203** [comment]
[test]

File: cico_setup.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ function run_functional_tests() {
 
     # Run the E2E tests against the running fabric8-ui container
     docker exec -t -e REFRESH_TOKEN=$REFRESH_TOKEN $CID bash -c \
-        "cd tests && WEBDRIVER_VERSION=2.37 DEBUG=true HEADLESS_MODE=true BASE_URL='https://prod-preview.openshift.io/' ./run_e2e_tests.sh"
+        "cd tests && WEBDRIVER_VERSION=2.37 DEBUG=true HEADLESS_MODE=true BASE_URL='http://${SERVER_IP}:8080' ./run_e2e_tests.sh"
```
Comment (rgarg1) [inline cico_setup.sh]: LGTM

Comment (Raunak1203) [reply cico_setup.sh]: @rgarg1 this was the temporary change to run tests against prod-preview, I changed it back.
We will be using cico_test_prod.sh to tests against production

**Comment by Raunak1203** [comment]
[test]

**Pull request merged**